hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -834,3 +835,48 @@ def delete_atom(self, handle: str, **kwargs) -> None:
                 )
 
             self._update_link_index([document], delete_atom=True)
+
+    def create_field_index(self, collection_name: str, indexes: tuple) -> str:
```

Comment:
Where is this method called? I ask because it expects "collection_name" which is a mongoDB specific concept. Although it's not a problem to have it here, it may be a problem to have the caller using it.

Reply:
Today this method is used by `LocalQueryEngine` in the `das-query-engine` repository. I understand your point, `collection_name` is just a convention, but it can be `table_name` or any other name that can describe it. Do you have a suggestion?